src/components/molecules/NavbarSchedule/NavbarSchedule.scss

```diff
@@ -0,0 +1,53 @@
+@import "scss/constants.scss";
+
+.nav-schedule {
+  position: relative;
+  background-color: rgba($white, 0.08);
+  height: $schedule-height;
+  padding: 0 50px 0 10px;
+  text-align: center;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  transition: transform 400ms $transition-function;
+  text-indent: 20px;
+
+  img {
+    height: 40px;
+  }
+
+  &:hover {
+    background-color: rgba($white, 0.14);
+    &:after {
+      opacity: 0.7;
+    }
+  }
+
+  &:after {
+    content: "";
+    position: absolute;
+    right: 20px;
+    top: 20px;
+    height: 20px;
+    width: 20px;
+    background-image: url(/icons/nav-party-logo-icon.png);
```

Comment:
Does this still work when merging the latest code in from `staging`? I believe we refactored a whole lot of the code for icons/images that used to be loaded from `public/`, so I expect this (and potentially any other places that have similar) are going to break.

We need to confirm this against the equivalent styles currently on `staging` before this is good to merge.